test/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote": true,
```

Comment:
it's interesting that this is single quote, but the src file has been updated to be all double quotes

Reply:
Good catch. I thin it's because I accidentally put it in the `/tests` folder.